app/views/orders/show.html.erb

```diff
@@ -1,28 +1,69 @@
 <p id="notice"><%= notice %></p>
 
-<p>
-  <strong>Order number:</strong>
-  <%= @order.order_number %>
-</p>
 
-<p>
-  <strong>Order date:</strong>
-  <%= @order.order_date %>
-</p>
+<div class="jumbotron">
+  
+  <div class="container">
+  
+      <div class="row col-xl-12" style="color:black">
+        <div class="col-sm-6">
+          <strong>Product Image</strong>
+          <p>
+            <%=image_tag((Product.find_by(params[@order.product_id]).pimage))%>
```

Comment:
What are params here??

Reply:
yesterday we had conversation about it and i dont have to fire quries from views.
in params i was trying to get product details from product_id column which is in order table, but i know this is wrong practice